### PR TITLE
Feature/iii 3416 Optimize styling giftbox

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Added

- more semantic html elements

### Changed

- styling of giftbox to resemble other modals like in advanced search

### Fixed

- title that wasn't ellipsing when too long

---
Ticket: https://jira.uitdatabank.be/browse/III-3416

Screenshot:
Schermafbeelding 2020-08-20 om 10 08 10

### CODE REVIEW

File: components/giftbox.vue
Comment (bertramakers) [inline components/giftbox.vue:176]: Nit: I don't think we need the browser prefixes for `box-shadow` anymore. See https://css-tricks.com/do-we-need-box-shadow-prefixes/ and https://caniuse.com/#feat=css-boxshadow

Worst case scenario someone with a _very_ old browser won't see the shadow, but it won't hurt the experience.

**Comment by bertramakers** [comment]
@simon-debruijn is there something still missing in this PR or can we convert it from draft to ready for review? All looks good to me 👍  But I can't approve as long as it's a draft.

**Comment by simon-debruijn** [comment]
> @simon-debruijn is there something still missing in this PR or can we convert it from draft to ready for review? All looks good to me 👍 But I can't approve as long as it's a draft.

Moved to Ready for review

**Review by bertramakers — APPROVED** [review]

**Pull request merged**
